package.json

```diff
@@ -35,6 +35,7 @@
     "@azure/identity": "^1.0.3",
     "@azure/keyvault-secrets": "^4.0.4",
     "@google-cloud/secret-manager": "^3.2.3",
+    "@ibm-cloud/secrets-manager": "^0.0.6",
```

Comment:
Looks like a newer version 0.1.0 is no available, might be worth bumping? 😄

Reply:
I'll take a look at bumping this and test it.

Reply:
So it looks like 0.1.0 release hasn't made it into npm yet so this can't land yet. 
We should get it in so I'll follow up with @IdanAdar to see when it will land in the registry.

Reply:
Published.

Reply:
Hey @Flydiverny Patch applied but the container has CVEs do I need to fix this?

Reply:
@No9 thanks for the update, no need to address the CVE. I was looking at it earlier and there's a fix out already. But it seems our pipeline is pulling an old base image.